app/company/models.py

```diff
@@ -115,6 +120,21 @@ class Meta:
         verbose_name=_('Phone number')
     )
 
+    person_in_charge = models.ForeignKey(
```

Comment:
Can't be null because you use this field in :

```
/home/rignonnoel/Documents/Code/TableauDeBord/app/floorMap/static/py/invoice_export.py in export_to_excel
    first_name=rent.company.person_in_charge.user.first_name,
```